### DISCUSSION TITLE
Export 0xxx liked string handler

### DISCUSSION THREAD

**Discussion by ChimingPhang** [react-csv/react-csv#345] [Ideas]
I suffer some issues like this:
001 will be formatted to 1 in CSV  file;
My suggestion is to add some code like the below:
<img width="567" alt="image" src="https://user-images.githubusercontent.com/3283625/186806188-d97b5919-c709-418d-95e1-e0faf0d568cf.png">

**Comment by davemjones** [comment]
I have a similar need for this as well.
